Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -2740,6 +2743,12 @@ private void LynchCycle()
                         if (lynched.PlayerRole == IRole.Prince & !lynched.HasUsedAbility) //can only do this once
                         {
                             SendWithQueue(GetLocaleString("PrinceLynched", lynched.GetName()));
+                            var ScapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
+                            if (ScapeGoat != null)
```

Comment:
Local variable names should be `snakeCase` in C#
```suggestion
                            var scapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
                            if (scapeGoat != null)
```